docs-user/_sidebar.md

```diff
@@ -5,6 +5,7 @@
   * [UI Tour](./guide-ui-tour.md)
   * [Stack samples and call trees](./guide-stack-samples-and-call-trees.md)
   * [Filtering call trees](./guide-filtering-call-trees.md)
+  * [Startup](./guide-startup.md)
```

Comment:
nit: Please use the same title as the page: `Profiling Firefox Startup`

Reply:
ok @julienw